uui/data/processing/views/ArrayListView.ts

```diff
@@ -171,7 +171,9 @@ export class ArrayListView<TItem, TId, TFilter = any> extends BaseListView<TItem
             row.isFocused = this.value.focusedIndex === index;
         });
 
-        if (all.checkableCount > 0) {
+        const isSelectAllSwitched = this.props.selectAll == undefined ? true : this.props.selectAll;
```

Comment:
don't copy-paste this for all views, move this to the BaseListView. And may be it better to name this - 'isSelectAllEnabled'?